SKILL.tsv

```diff
@@ -5734,302 +5734,302 @@ SKILL_20151102_005733	$[Merkabah] petrifies [Cursed] enemies for 3 seconds.
+SKILL_20151224_005805	{#993399}{ol}[Magical]{/}{/}{nl}Put on shoes made of mixed hair and blood. Walk while leaving bloody footprints in the ground, which curses any enemies who come into contact with them.
+SKILL_20151224_005806	Maximum Kurdaitcha duration: #{CaptionRatio}#second{nl}Moving speed reduction: 15{nl}{#339999}{ol}[Curse]{/}{/} Duration: 7 second
+SKILL_20151224_005807	{#993399}{ol}[Magical] - [Dark]{/}{/}{nl}Summon a pillar of darkness, which causes enemies in contact to suffer continuously.
+SKILL_20151224_005808	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio}#%{nl}Duration: #{CaptionTime}#second{nl}#{CaptionRatio2}#Additional damage per second: 
+SKILL_20151224_005809	{#993399}{ol}[Magical] - [Dark]{/}{/}{nl}Summon evil spirits from the death of enemies. Summoned spirits remain in the field damage enemies in contact with them.
+SKILL_20151224_005810	{#993399}{ol}[Magical] - [Dark]{/}{/}{nl}Summon spirits to protect the caster by surrounding them.
+SKILL_20151224_005811	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio}#%{nl}Duration: 30 second
+SKILL_20151224_005812	{#993399}{ol}[Magical] - [Dark]{/}{/}{nl}Summon demons of hatred to attack enemies. Prioritize the enemy hit by this magic and hit with additional damage by holy attacks.
+SKILL_20151224_005813	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio}#%{nl}Stigma duration: 5 second
+SKILL_20151224_005814	Evil Sacrifice
+SKILL_20151224_005815	{#993399}{ol}[Magical]{/}{/}Direct spirits to a location, and dedicate enemies in range as a sacrifice. Summoned spirits will move towards that area.
+SKILL_20151224_005816	Install a shield to protect themselves from enemies and long-ranged attacks. Enemies without a target will attack the pavise instead.
+SKILL_20151224_005817	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio}#%{nl}Throw count: #{CaptionRatio2}#{nl}Caltrop duration: #{CaptionTime}#second{nl}Decrease enemy moving speed by 10 for 5 second
+SKILL_20151224_005818	Pick up stone ammunition used for Stone Shot.
+SKILL_20151224_005819	Attack with a crossbow in while moving in fast speed. Attacking speed is raised while additional damage is applied to basic attacks.
+SKILL_20151224_005820	Duration: #{CaptionTime}#second{nl}Increase attacking speed by 30%{nl}Additional damage: #{CaptionRatio}#%
+SKILL_20151224_005821	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio2}#%{nl}Explosion count: #{CaptionRatio}#
+SKILL_20151224_005822	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio2}#%{nl}Install count: #{CaptionRatio}#
+SKILL_20151224_005823	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionTime}#%{nl}Arrow count: #{CaptionRatio}#{nl}Install range: #{CaptionRatio2}#{nl}Consume 2 wires
+SKILL_20151224_005824	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}The companion bites and hold the enemy in place. The bitten enemy will have their physical defense reduced while critical hit rate bonus is applied against the enemy.
```

Comment:
The bitten enemy will have their physical defense reduced while critical hit rate bonus is applied against the enemy.

to

The bitten enemy will have its physical defense reduced while critical chance bonus is applied against the enemy.